From 15ee1c2e59dc8d563b8db8035836b68a5a82f6e9 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Tue, 17 Oct 2023 22:22:27 +0200
Subject: [PATCH] dbus: Remove at_console filtering

This is long deprecated, remove it since it prohibits communication with
repowerd.
---
 data/dbus-configs/com.canonical.Unity.Screen.conf | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/data/dbus-configs/com.canonical.Unity.Screen.conf b/data/dbus-configs/com.canonical.Unity.Screen.conf
index 55d2c7b..7bb4cb8 100644
--- a/data/dbus-configs/com.canonical.Unity.Screen.conf
+++ b/data/dbus-configs/com.canonical.Unity.Screen.conf
@@ -9,11 +9,6 @@
 	</policy>
 
 	<policy context="default">
-		<allow send_destination="com.canonical.Unity.Screen"
-		       send_interface="org.freedesktop.DBus.Introspectable"/>
-	</policy>
-
-	<policy at_console="true">
 		<allow send_destination="com.canonical.Unity.Screen"/>
 	</policy>
 </busconfig>
-- 
2.42.0

